azure/Guidelines.md

```diff
@@ -747,18 +747,70 @@ The ability to retry failed requests for which a client never received a respons
+
+:white_check_mark: **DO** return a `202-Accepted` status code from the request that initiates a stepwise LRO if the processing of the operation was successfully initiated.
+
+:white_check_mark: **DO** perform as much validation of the initial request as practical and return an error response when appropriate.
+
+:white_check_mark: **DO** return the operation-results URL in the `Operation-Location` response header.
+
+:white_check_mark: **DO** support the `get` method on the operation-results endpoint that returns a `200-OK` response with a response body that indicates whether the operation has completed and completion status (success, failed) with sufficient error information to diagnose failures.
+
+:white_check_mark: **DO** include a field in the operation-results named `status` to contain the operation status. This field should be a string with well defined values that must include "Succeeded", "Failed", and "Canceled".
+
+:white_check_mark: **DO** include a field in the operation-results named `error` to contain error information -- minimally `code` and `message` fields -- when an operation fails.
+
+:white_check_mark: **DO** support return of the operation-results for some documented period of time, a minimum of 24 hours, after the operation has completed.
+
+:white_check_mark: **DO** include a `Retry-After` header in the response to the initiating request and requests to the operation-results endpoint. The value of this header should be an integer number of seconds to wait before making the next request to the operations-results URL.
+
+:heavy_check_mark: **YOU MAY** support a `get` method on the operation-results collection endpoint that returns a list of all recently initiated operations.
+
+:no_entry: **DO NOT** return any other `2xx` status code from the initial request of a stepwise LRO -- return `202-Accepted` and an operations-result URL even if processing was completed before the initiating request returns.
+
+:no_entry: **DO NOT** return any data in the response body of a `202-Accepted` response.
+
+Previous Azure guidelines specified "Azure-AsyncOperation" as the name of the response header containing the operation-results URL.
+
+:white_check_mark: **DO** return **both** `Azure-AsyncOperation` and `Operation-Location` headers if your service previously returned `Azure-AsyncOperation`, even though they are redundant, so that existing clients will continue to operate.
 
 :white_check_mark: **DO** return the same value for **both** headers.
 
-:white_check_mark: **DO** look for **both** HEADERS in client code, preferring the `Operation-Location` version.
+:white_check_mark: **DO** look for **both** headers in client code, preferring the `Operation-Location` header.
```

Comment:
This is client guideline; not a service guideline. If you want to say something about this, then I'd merge it with the previous bullet and I'd give better guidance. Specifically, clients should look for 'operation-location' first and if not present, then look for 'azure-asyncoperation'.

Reply:
Remove "white_check_mark".